### PR TITLE
docs: update config-default.yaml

### LINKED ISSUE #6969: docs: As a user, I want to specify different registries for different routes
### Description

https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L304-L317
I found that the registries supported in our document are `dns`, `eureka`, `consul-kv` and `nacos`, but now there are only `dns` and `eureka` in config-default.yaml file, so I think it needs to add `consul-kv` and `nacos` to config-default.yaml

### PR DESCRIPTION
**Pull request by bin-ya**
### Description

Fixes #6969

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: conf/config-default.yaml [hunk 1]
```diff
@@ -315,6 +315,83 @@ etcd:
...
+#        fail_timeout: 1           # default 1 ms
+#        weight: 1                 # default 1
+#        max_fails: 1              # default 1
+#    dump:                         # if you need, when registered nodes updated can dump into file
+#       path: "logs/consul_kv.dump"
+#       expire: 2592000      # unit sec, here is 30 day
+#  kubernetes:
+#    service:
+        # apiserver schema, options [http, https]
+#      schema: https #default https
```
Comment (starsz) [inline conf/config-default.yaml:355]:
```suggestion
#      schema: https # apiserver schema, options [http, https], default https
```

File: conf/config-default.yaml [hunk 2]
```diff
@@ -315,6 +315,83 @@ etcd:
...
+#
```
Comment (starsz) [inline conf/config-default.yaml:356]: Redundant line.

File: conf/config-default.yaml [hunk 3]
```diff
@@ -315,6 +315,83 @@ etcd:
...
+        # apiserver host, options [ipv4, ipv6, domain, environment variable]
+#      host: ${KUBERNETES_SERVICE_HOST} #default ${KUBERNETES_SERVICE_HOST}
```
Comment (starsz) [inline conf/config-default.yaml:358]: Ditto.

File: conf/config-default.yaml [hunk 4]
```diff
@@ -315,6 +315,83 @@ etcd:
...
+#
+        # apiserver port, options [port number, environment variable]
+#      port: ${KUBERNETES_SERVICE_PORT}  #default ${KUBERNETES_SERVICE_PORT}
+#
+#    client:
+        # serviceaccount token or token_file
+#      token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
```
Comment (starsz) [inline conf/config-default.yaml:365]:
```suggestion
#      token_file: path/to/token   # path of serviceaccount token_file
```

**Review by membphis — APPROVED** [review]

File: conf/config-default.yaml [hunk 5]
```diff
@@ -315,6 +315,70 @@ etcd:
...
+#      - "http://127.0.0.1:8600"
+#    prefix: "upstreams"
+#    skip_keys:                    # if you need to skip special keys
+#      - "upstreams/unused_api/"
+#    timeout:
+#      connect: 1000               # default 2000 ms
+#      read: 1000                  # default 2000 ms
+#      wait: 60                    # default 60 sec
+#    weight: 1                     # default 1
+#    fetch_interval: 5             # default 3 sec, only take effect for keepalive: false way
```
Comment (starsz) [inline conf/config-default.yaml:340]: `fetch_interval: 5` , default 5 or 3 ？

File: conf/config-default.yaml [hunk 6]
```diff
@@ -315,6 +315,70 @@ etcd:
...
+#      #- ^my-[a-z]+$
+#      # only save endpoints with namespace not match one of [default, ^my-[a-z]+$ ]
+#      #not_match:
+#      #- default
+#      #- ^my-[a-z]+$
+#    # kubernetes discovery plugin support use label_selector
+#    # for the expression of label_selector, please refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/labels
+#    label_selector: |-
+#      first="a",second="b"
+#      # default 10s
```
Comment (starsz) [inline conf/config-default.yaml:381]: What's this? Which value defaults to` 10s`?

**Pull request closed without merging**